### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by MaksymKos**
- [DEMO LINK](https://MaksymKos.github.io/layout_catalog/)
- [TEST REPORT LINK](https://MaksymKos.github.io/layout_catalog/report/html_report/)
require some help with tests

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,12 +1,309 @@
...
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
```
Comment (IvanFesenko) [inline src/index.html:28]: don`t forget to add empty line between sibling tags
2023-05-15_15h04_52

File: src/index.html [hunk 2]
```diff
@@ -1,12 +1,309 @@
...
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">
+              Reviews: 5
+            </p>
+          </div>
+          <div class="card__price">
+            <div class="card__price-world">Price:</div>
+            <div class="card__price-number">$2,199</div>
```
Comment (IvanFesenko) [inline src/index.html:79]: Do not use div for text content. use semantic tag instead

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    height: 134px;
+    display: block;
+    margin: 0 auto;
+
+    transition: transform 400ms;
+
+    &:hover {
+      transform: scale(120%);
+      border: 3px $card-border-color solid;
+    }
```
Comment (IvanFesenko) [inline src/styles/blocks/card.scss:19]: Thats out of design

File: src/styles/blocks/card.scss [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $text-color-black;
+
+    margin-top: 40px;
+  }
+
+  &__product-code {
+    font-weight: 400;
```
Comment (IvanFesenko) [inline src/styles/blocks/card.scss:32]: ` font-weight` by default is 400 - redudant. Fix all similar cases

File: src/styles/blocks/card.scss [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+      font-weight: 400;
+      font-size: 12px;
+      color: $text-color-light;
+    }
+
+    &-number {
+      font-weight: 700;
+      font-size: 16px;
+      color: $text-color-black;
+    }
```
Comment (IvanFesenko) [inline src/styles/blocks/card.scss:70]: Try use more logical names: `description` and `value` for example
Also `world` - means `Світ`

File: src/styles/reset.scss [hunk 6]
```diff
@@ -0,0 +1,10 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
```
Comment (IvanFesenko) [inline src/styles/reset.scss:5]: font-family move to html tag
reset margin and paddings only for those tags you need
```suggestion
```

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Check my comments

File: src/styles/blocks/card.scss (hunk 3 above)
Comment (MaksymKos) [reply src/styles/blocks/card.scss:19]: Знімок екрана 2023-05-15 о 16 27 29

Comment (MaksymKos) [reply src/styles/blocks/card.scss:19]: According to tests and this file which I found in rep, I made this :hover

**Review by etojeDenys — APPROVED** [review]
lgtm

**Pull request closed without merging**